tutorials/variational-quantum-eigensolver/vqe.ipynb

```diff
@@ -187,33 +177,33 @@
    "id": "ac6f36e3",
    "metadata": {},
    "source": [
-    "## Step 2: Optimize problem for quantum execution."
+    "## Step 2: Optimize problem for quantum execution"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "ed01c675-6506-4779-bf71-74f0de9212fb",
    "metadata": {},
    "source": [
-    "To reduce the total job execution time, Qiskit Runtime V2 primitives only accept circuits (ansatz) and observables (Hamiltonian) that conforms to the instructions and connectivity supported by the target system (referred to as instruction set architecture (ISA) circuits and observables, respectively)."
+    "To reduce the total job execution time, Qiskit primitives only accept circuits (ansatz) and observables (Hamiltonian) that conform to the instructions and connectivity supported by the target system (referred to as instruction set architecture (ISA) circuits and observables)."
    ]
   },
   {
    "cell_type": "markdown",
    "id": "3390069d-572c-472c-abb5-9cde12fd82a2",
    "metadata": {},
    "source": [
-    "### ISA Circuit"
+    "### ISA circuit"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "ad6ddd99-b680-4ac4-b2d8-c0ac6266e6e8",
    "metadata": {},
    "source": [
-    "We can schedule a series of [`qiskit.transpiler`](https://docs.quantum-computing.ibm.com/api/qiskit/transpiler) passes to optimize our circuit for a selected backend and make it compatible with the instruction set architecture (ISA) of the backend. This can be easily done using a preset pass manager from `qiskit.transpiler` and its `optimization_level` parameter.\n",
+    "Schedule a series of [`qiskit.transpiler`](https://docs.quantum-computing.ibm.com/api/qiskit/transpiler) passes to optimize the circuit for a selected backend and make it compatible its ISA. This can be easily done with a preset pass manager from `qiskit.transpiler` and its `optimization_level` parameter.\n",
```

Comment:
```suggestion
    "Schedule a series of [`qiskit.transpiler`](https://docs.quantum-computing.ibm.com/api/qiskit/transpiler) passes to optimize the circuit for a selected backend and make it compatible with the backend's ISA. This can be easily done with a preset pass manager from `qiskit.transpiler` and its `optimization_level` parameter.\n",
```